Frameworks/UIKit.Xaml/ActivityIndicatorView.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ActivityIndicatorView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root">
+    <!-- TODO: Update property names in path for binding once they have been updated in the header -->
+    <ProgressRing x:Name="progressRing"
+                  IsActive="{Binding ElementName=Root, Path=ActivityIndicatorView_IsActive, Mode=OneWay}"
+                  Foreground="{Binding ElementName=Root, Path=ActivityIndicatorView_Foreground, Mode=OneWay}"
+                  Height="{Binding ElementName=Root, Path=ActivityIndicatorView_Height, Mode=OneWay}"
+                  Width="{Binding ElementName=Root, Path=ActivityIndicatorView_Width, Mode=OneWay}"
```

Comment:
I'm confused by these - why not just bind to Width/Height of the containing Grid?